src/main/java/org/folio/circulation/support/HttpFailure.java

```diff
@@ -4,4 +4,6 @@
 
 public interface HttpFailure {
   void writeTo(HttpServerResponse response);
+
+  String getReason();
```

Comment:
What is the exposed reason used for?

Reply:
it used for logging

Reply:
Is that for logging whether a check in log record has been created?

Reply:
Yes, it used to log a check-in log failure, if any.

Reply:
Could an alternative be to use the `toString` method and override that for each kind of failure?

Reply:
Yes it can. But `toString` is not a part of `HttpFailure` contract so it won't require `toString` to be overridden and new implementations of `HttpFailure` can use default toString with no useful info.

Reply:
Ok. I think that makes the difference that one requires a method to be overridden and the other does not either. Either might not produce useful information.

I think I should resolve this comment. I would prefer the interface wasn't expanded, to expose internal details, however I don't have a good enough argument against it.